docs/api/routing.rst

```diff
@@ -168,6 +168,24 @@ For ASGI apps, the responder must be a coroutine function::
     async def on_post(self, req, resp):
         pass
 
+
+QUERY method
```

Comment:
I think this section might be too excessive for a single HTTP verb, we don't document other verbs on this level either. Let's write a concise news fragment instead.